internal/tests/e2e_sc_happy_path.go

```diff
@@ -0,0 +1,431 @@
+package tests
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	scTypes "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	scClient "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+
+	bucTypes "github.com/kyma-project/kyma/components/binding-usage-controller/pkg/apis/servicecatalog/v1alpha1"
+	bucClient "github.com/kyma-project/kyma/components/binding-usage-controller/pkg/client/clientset/versioned"
```

Comment:
our goal was to find problems with the service catalog, do we really need to create sbu's in this test?

Reply:
addressed by adding new flag. This can be simply disabled